src/extensions/mountainview/MVSortingView/MVSortingView.tsx

```diff
@@ -76,33 +76,77 @@ export const openViewsReducer: React.Reducer<View[], OpenViewsAction> = (state:
     else return state
 }
 
-const MVSortingView: FunctionComponent<SortingViewProps> = (props) => {
+const area = (a: 'north' | 'south') => {
+    return a
+}
+
+const MVSortingViewWithCheck: FunctionComponent<SortingViewProps> = (props) => {
```

Comment:
Create a parent component that does the preloading.
We want this to be slick -- so we DO show the child component (the GUI) but we
(a) prevent interaction
(b) do not send the updated recording/sorting until after it has been computed

The difficult case is when the user selects a new preprocessing (bandpass filter for example), because then we need to start the preprocessing, but the child component should still hold on to the old recording until the preloading is complete. That way the user doesn't see a blank screen.

It took me a while to get this right... but it's working nicely now.